content/docs/reference/routes/from.mdx

```diff
@@ -16,21 +17,62 @@ import TabItem from '@theme/TabItem';
 
 ## Summary
 
-**From** is the externally accessible URL for the proxied request.
+The **From** route is the externally accessible URL for a proxied HTTP request.
```

Comment:
I don't think we should use the phrase "From route." A route must have both a from URL and a to URL. There aren't "from routes" or "to routes".

Reply:
So a "route" consists of a From and To URL. I agree here. I thought it made sense to distinguish which one of these we're referring to. "From route" could be "From URL".